src/app/space/[did]/root/[cid]/shard/[shard]/page.tsx

```diff
@@ -28,23 +29,38 @@ interface PageProps {
   }
 }
 
+const isCARLink = (link: Link): link is CARLink => link.code === CAR.codec.code
+
 export default function ItemPage ({ params }: PageProps): JSX.Element {
   const [{ client, spaces }] = useW3()
   const spaceDID = decodeURIComponent(params.did)
   const space = spaces.find(s => s.did() === spaceDID)
   const root = parseLink(params.cid)
-  const shard = parseLink(params.shard)
+  const shard = parseLink(params.shard).toV1()
 
   const storeKey = `/space/${spaceDID}/store/get?link=${shard}`
-  const store = useSWR<StoreGetSuccess|undefined>(storeKey, {
+  const store = useSWR<{ size: number } | undefined>(storeKey, {
     fetcher: async () => {
-      if (!client || !space) return
+      if (!client || !space || !(isCARLink(shard))) return
```

Comment:
Does it matter if it's not a CAR shard?

Reply:
Yeah, `client.capability.store.get(shard)` takes a `CARLink`. Why this wasn't failing before I can't explain; it's red in my editor. 😕